sciencebox/values.yaml

```diff
@@ -587,6 +583,120 @@ swan:
     #       cert:
     #       key:
 
+## cbox web
+cernbox-web:
+  initContainers:
+    - name: install-web
+      image: busybox
+      command: ["/bin/sh", "-c"]
+      args: 
+        - wget -O /var/www/web/web-tar.gz https://github.com/cernbox/web-release/releases/download/v0.0.9/web.tar.gz;
+          tar -xf /var/www/web/web-tar.gz -C /var/www/web/;
+          rm -rf /var/www/web/web.tar.gz;
+          wget -O /var/www/web/theme.json https://new.cernbox.cern.ch/cernbox/theme-cernbox-0.1.7/theme.json;
```

Comment:
Why don't you just download the tar of the theme? It would guarantee you don't miss any file

Reply:
https://github.com/cernbox/web-extensions/releases/tag/theme-cernbox%2Fv0.1.7

Reply:
DONE